### DISCUSSION TITLE
Letters and numbers after some Muse Sounds instruments names

### DISCUSSION THREAD

**Discussion by xman2156** [musescore/MuseScore#15181] [Q&A] [answered]
What are the "a6" and "a3" markings after the Horns and Trombones Muse Sounds names. They've been there since the beta, but I figured they would be gone by the official release.
![image](https://user-images.githubusercontent.com/34029777/207707433-f793ba1f-314a-4d10-bd5a-cac986ef0be8.png)
![image](https://user-images.githubusercontent.com/34029777/207707468-4ab33224-df10-4c64-837f-9d9cb4d9da9e.png)

**Comment by ghost** [comment, marked as answer]
a6 = 6 horns (in unison), a3 = 3 horns (in unison).
